megamek/src/megamek/common/Jumpship.java

```diff
@@ -181,6 +209,68 @@ public static TechAdvancement getJumpSailTA() {
                 .setAvailability(RATING_E, RATING_E, RATING_D, RATING_D)
                 .setStaticTechLevel(SimpleTechLevel.ADVANCED);
     }
+    
+    public String getCritDamageString() {
+        String toReturn = super.getCritDamageString();
```

Comment:
Ideally this would use a `StringBuilder`.

Reply:
Fixed. Also updated super methods.